snapshot/overlay/overlay.go

```diff
@@ -297,11 +322,29 @@ func (a *activeDir) mounts(c *cache) ([]containerd.Mount, error) {
 			},
 		}, nil
 	}
-	options := []string{
-		fmt.Sprintf("workdir=%s", filepath.Join(a.path, "work")),
-		fmt.Sprintf("upperdir=%s", filepath.Join(a.path, "fs")),
-		fmt.Sprintf("lowerdir=%s", strings.Join(parents, ":")),
+	var options []string
+
+	if _, err := os.Stat(filepath.Join(a.path, "fs")); err == nil {
+		options = append(options,
+			fmt.Sprintf("workdir=%s", filepath.Join(a.path, "work")),
+			fmt.Sprintf("upperdir=%s", filepath.Join(a.path, "fs")),
```

Comment:
I think would be great if we converted the names of these directories to match overlay terminology.